modules/ROOT/pages/fd-project-history.adoc

```diff
@@ -0,0 +1,30 @@
+= Discover What’s Changed in a Project
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords:
+
+:imagesdir: ../assets/images
+
+You can see a history of changes you make to a project in Flow Designer.
+The saved states are listed by the ID of the user who made the change, time, date, and activity. The activity name is based on the action you performed at save time.
+
+Being able to view a saved state of a project is useful if you want to:
+
+* Undo an inadvertent change you made to a project and revert back to the working state.
+* Review changes made by other users on the project.
+* Retrieve older values of some project configurations.
+
+
+== Procedure
+
+To restore to a previous version:
```

Comment:
Shouldn't this be "To restore a previous version of your project:"? Restore is a transitive verb, so requires an object.

Reply:
Also, I see "state" used in the steps and in the result of the procedure. What is being restored: a state or a version?

Reply:
Yes to the first item.
To the second item - yes it is a version. However, we have been calling these 'versions' saved states. I see your point tho. I changed the wording to address your comment.